must-gather/collection-scripts/gather-data-science-pipelines

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+datasciencepipelines=$(oc get DataSciencePipelinesApplication --all-namespaces -o=jsonpath='{range .items[*]}{.metadata.namespace}{"\n"}{end}')
+
+# Get logs from all Data Science Pipelines Application namespaces
+for i in $datasciencepipelines;
+do
+# Get pod logs for all the pods in Data Science Pipelines Application namespaces
+oc adm inspect namespace/"$i" --dest-dir=must-gather || echo "Error getting logs from $i"
+
+done
+
+## Get Operator pod logs
```

Comment:
shouldn't this be for the DSP Operator, not ODH operator?

Reply:
Done, updated to have this for DSP Operator.

Reply:
Updating to take must gather for DSP Operator out, per discussion with @VaishnaviHire . If any application is deployed using kfdef we already collect the pod logs, hence this is not required.